api/routes/jwtAuth.ts

```diff
@@ -0,0 +1,76 @@
+import pool from '../configDB/config';

```

Comment:
Powinniśmy mieć folder User i tam zrobić endpointy na to, dzięki czemu bedziemy wszystko mieć ładnie poukładane i wiadomo gdzie co będzie